src/analyzer/protocol/tcp/TCP.cc

```diff
@@ -478,8 +478,8 @@ static void syn_weirds(TCP_Flags flags, TCP_Endpoint* endpoint, int data_len)
 void TCP_Analyzer::UpdateInactiveState(double t,
 			TCP_Endpoint* endpoint, TCP_Endpoint* peer,
 			uint32_t base_seq, uint32_t ack_seq,
-			int len, int is_orig, TCP_Flags flags,
-			int& do_close, int& gen_event)
+			int len, bool is_orig, TCP_Flags flags,
+			bool& do_close, bool& gen_event)
```

Comment:
Note for future work: non-const pass-by-reference would be good to replace with pass-by-pointer.